packages/cms-cli/commands/hubdb.js

```diff
@@ -71,6 +77,43 @@ function configureHubDbCreateCommand(program) {
   addConfigOptions(program);
 }
 
+function configureHubDbImportCommand(program) {
+  program
+    .version(version)
+    .description('Import HubDB table rows')
+    .arguments('<tableId> <src>')
+    .action(async (tableId, src, command = {}) => {
```

Comment:
The last argument passed to `action()` is always the Commander instance.  Thus `program === command`.
I've been preferring to treat the parameters explicitly and just use `program`.  Also if your last param is optional you can test it against `program`.